packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -110,6 +110,8 @@ html.interface-interface-skeleton__html-container {
 .interface-interface-skeleton__left-sidebar {
 	@include break-medium() {
 		border-right: $border-width solid $gray-200;
+		overflow: hidden; // prevent overscroll
+		padding-bottom: 24px; // compensate for breadcrumb footer
```

Comment:
The height is actually 25px. 24px + 1px border top. 

How about using: `$button-size-small + $border-width` ?

Reply:
Ah yes, should have thought that we had these in variables, thanks!